spec/bolt/inventory/inventory2_spec.rb

```diff
@@ -665,4 +665,45 @@ def common_data(transport)
       end
     end
   end
+
+  context 'with localhost' do
```

Comment:
We should have a test to ensure this works without anything named localhost too probably.

Reply:
I'm still not sure where we're testing that the special localhost target works even if the user has no node named localhost in their inventory(which is the more common case)